service/src/main/resources/application.yml

```diff
@@ -14,6 +14,8 @@ env:
     resourceId: ${SAM_RESOURCE_ID:axonserver-dev}
   calhoun:
     basePath: ${CALHOUN_ADDRESS:https://terra-calhoun.api.verily.com}
+  cromwell:
+    basePath: http://localhost:8000
```

Comment:
this needs to take a config first, like `${CROMWELL_ADDRESS:http://localhost:8000}`